web/app/Makefile

```diff
@@ -0,0 +1,45 @@
+export BABEL_ENV ?= development
+
+.PHONY: build
+build:
+	@# Build the assets.
+	yarn && yarn webpack
+
+.PHONY: dev
+dev:
+	@# Run a dev server, to do development with.
+	yarn webpack-dev-server &
+	cd .. && ../bin/go-run . --webpack-dev-server=http://localhost:8080
+
+.PHONY: run
+run: build
+	@# Run a local server (sans. reloading).
+	cd .. && ../bin/go-run .
+
+.PHONY: test
+test:
+	@# Run the tests
+	yarn karma start --single-run
+
+.PHONY: help
```

Comment:
the `make help` output looks a bit wonky to me - some extra newlines and then weird formatting in "run: build" and "build"

```
$ make help
usage: make [target] ...






dev:
	 Run a dev server, to do development with.


help:
	 Output all targets available.


run: build
	 Run a local server (sans. reloading).


test:
	 Run the tests

build
	 Build the assets.
```